app/views/fileStatus.scala.html

```diff
@@ -59,9 +59,15 @@ <h2 class="govuk-error-summary__title" id="error-summary-title">
                                         </div>
                                     </div>
 
-                                    <a href="@routes.ReviewController.index(consignmentId)" role="button" draggable="false" class="govuk-button govuk-button--primary" data-module="govuk-button">
-                                        @messages("filestatus.success.confirm")
-                                    </a>
+                                    <!-- Buttons -->
+                                    <div class="govuk-form-group">
+                                        <a href="@routes.DashboardController.index" role="button" draggable="false" class="govuk-button govuk-button--secondary" data-module="govuk-button">
```

Comment:
Are we clear about what the consequences will be if a user cancels the processing whilst its in progress? Should check with Sam and Suzanne about this.